src/ui/src/components/core/root/CoreRoot.vue

```diff
@@ -107,12 +135,12 @@ watch(activePageId, (newPageId) => {
 	changePageInHash(pageKey);
 });
 
-type ParsedHash = {
+type UrlParams = {
```

Comment:
I understand `ParsedHash` isn't an ideal name, but `UrlParams' makes me think of querystring, which this isn't.

@raaymax suggested `UrlHashParams` can we change?